### DISCUSSION TITLE
Gas Consumption Octopus Mini

### DISCUSSION THREAD

**Discussion by nivek1612** [BottlecapDave/HomeAssistant-OctopusEnergy#1102] [Q&A]
I have the octopus mini and it’s working well with one exception the current gas consumption 
sensor.octopus_energy_gas_{{METER_SERIAL_NUMBER}}_{{MPRN_NUMBER}}_current_consumption

Is always 0. Is this a know issues ? Or did I miss a setting to enable this 
All the other gas entities are working as expected

**Comment by BottlecapDave** [comment]
Current consumption is the delta value from the last update. Gas meters usually only update about once every 30 minutes. You're probably after the accumulative consumption sensor.
